ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Envelope.scala

```diff
@@ -6,6 +6,7 @@ package com.daml.ledger.participant.state.kvutils
 import java.util.zip.{GZIPInputStream, GZIPOutputStream}
 
 import com.daml.ledger.participant.state.kvutils.{DamlKvutils => Proto}
+import com.daml.ledger.participant.state.kvutils.{DamlState => ProtoState}
```

Comment:
Perhaps you can just stop aliasing these. I think I did this to make things clearer but because it's different to every other file, it's probably not very helpful.